app/views/publishers/email_verified.html.slim

```diff
@@ -1,3 +1,49 @@
+- if @taken_youtube_channel
+  javascript:
+    (function() {
+      function showYoutubeChannelTakenModal() {
+        var modal = document.getElementById('taken_youtube_channel_modal');
+        classie.add(modal, 'md-show');
+
+        var closeModal = document.getElementById('close_modal');
+        closeModal.addEventListener('click', function() {
+          hideYoutubeChannelTakenModal();
+        });
+        document.addEventListener('keyup', function(e) {
+          if (e.keyCode === 27) hideYoutubeChannelTakenModal();
+        });
+      }
+
+      function hideYoutubeChannelTakenModal() {
+        var modal = document.getElementById('taken_youtube_channel_modal');
+        classie.remove(modal, 'md-show');
+      }
+
+      window.addEventListener('load', function() {
+        showYoutubeChannelTakenModal();
+      });
+    })();
+
+  .md-container#taken_youtube_channel_modal
```

Comment:
Underscores have been used extensively elsewhere (especially on the dashboard) for element ids while dashes have been used class names. I strongly agree that dashes should be used for classes (b/o conventions), but I'm torn re: ids because Rails itself uses underscores for auto-generated element ids. Can we discuss this post-release and then, if we decide to switch over to dashes for ids, make the change in a single comprehensive commit?